### DISCUSSION TITLE
Gallery without user interaction. It's possible?

### DISCUSSION THREAD

**Discussion by gpsblues** [GetPublii/Publii#1549] [Help] [answered]
Hi everyone.

Do you know of an easy way to disable user clicking on galleries. I would like to use the gallery only for a quick layout of the images. I would like to view the gallery, but without any user interaction.

Thank you

**Comment by bobmitro** [comment, marked as answer]
Easy and quick way: add the following CSS rule using Custom CSS tool:
```
.gallery__item {
   pointer-events: none;
}
```

**Reply by michaelfluegge** [reply]
@bobmitro , one question: When using your CSS, the new [Lightboxes](https://getpublii.com/blog/publii-gallery-plugins.html) does not work, too, or? If I use pointer-events: none in Safari, the Lightbox does not appear...

**Comment by gpsblues** [comment]
Thanks so much Bob!
I was lost in JS functions and the solution was simple CSS. Beautiful!
